### PR TITLE
Chiclet: Styling updates

### PR DESCRIPTION
**Pull request by staciecarroll**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

This PR makes the following changes: 

- Medium Chiclet is now styled according to redlines
- Chiclet examples are now styled according to redlines
- xSmall Chiclet url is no longer cut off

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10044)

### CODE REVIEW

File: packages/experiments/package.json [hunk 1]
```diff
@@ -49,8 +49,7 @@
     "react-highlight": "0.10.0",
     "react-hooks-testing-library": "^0.5.0",
     "react-test-renderer": "^16.3.0",
-    "sinon": "^4.1.3",
-    "@uifabric/build": "^7.0.0"
```
Comment (natalieethell) [inline packages/experiments/package.json]: I believe we need to keep this

File: packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx [hunk 2]
```diff
@@ -81,6 +83,9 @@ export const getStyles = (styleProps: IChicletCardStyleProps): IChicletCardStyle
         // @todo: change values for height and width according to redline
```
Comment (natalieethell) [inline packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx]: Do we need the todo anymore? Are height and width provided in the redlines?

Comment (natalieethell) [reply packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx]: Same question for the icon styling above

Comment (staciecarroll) [reply packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx]: No, I will delete the todos that are complete!

File: packages/experiments/src/components/Chiclet/ChicletCard.base.tsx [hunk 3]
```diff
@@ -49,13 +48,7 @@ export class ChicletCardBase extends React.Component<IChicletCardProps, {}> {
...
           React.cloneElement(preview, { className: css(preview.props.className, customPreviewStyling) })
         ) : (
-          <Image
-            width={PREVIEW_IMAGE_WIDTH}
-            height={PREVIEW_IMAGE_HEIGHT}
-            src={image}
-            role="presentation"
-            alt={imageAlt ? imageAlt : undefined}
-          />
+          <img width={'auto'} height={'100%'} src={image} role="presentation" alt={imageAlt ? imageAlt : undefined} />
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/Chiclet/ChicletCard.base.tsx]: If this `` has meaningful content in some or all cases, consider removing `role="presentation"`. See: https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/presentation/PresentationRoleExamples.html

File: packages/experiments/src/components/Chiclet/ChicletCard.base.tsx [hunk 4]
```diff
@@ -49,13 +48,7 @@ export class ChicletCardBase extends React.Component<IChicletCardProps, {}> {
...
           React.cloneElement(preview, { className: css(preview.props.className, customPreviewStyling) })
         ) : (
-          <Image
-            width={PREVIEW_IMAGE_WIDTH}
-            height={PREVIEW_IMAGE_HEIGHT}
-            src={image}
-            role="presentation"
-            alt={imageAlt ? imageAlt : undefined}
-          />
+          <img width={'auto'} height={'100%'} src={image} alt={imageAlt ? imageAlt : undefined} />
```
Comment (natalieethell) [inline packages/experiments/src/components/Chiclet/ChicletCard.base.tsx]: Is there a reason this image styling is different than the one in ChicletCard? Sorry if we already discussed this and I missed it!

**Comment by natalieethell** [comment]
It looks like there are just a few snapshots that need to be updated! And to resolve the conflict, try merging with the latest master

File: packages/experiments/src/components/Chiclet/ChicletCard.styles.ts [hunk 5]
```diff
@@ -1,10 +1,14 @@
 import { normalize } from '../../Styling';
 import { IChicletCardStyleProps, IChicletCardStyles } from './ChicletCard.types';
+import { FontWeights } from '@uifabric/styling';
```
Comment (Vitalius1) [inline packages/experiments/src/components/Chiclet/ChicletCard.styles.ts]: This import should be combined with the one from line 1

File: packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import { normalize, getGlobalClassNames } from '../../Styling';
 import { IChicletCardStyleProps, IChicletCardStyles } from './ChicletCard.types';
+import { FontWeights } from '@uifabric/styling';
```
Comment (Vitalius1) [inline packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx]: Combine the import with the one from line 1

**Pull request merged**